Makefile

```diff
@@ -1,17 +1,57 @@
 # These assume you hav Docker installed and running
-
-all: build test
+SHELL = bash
+all: build setupelk
 
 build:
-	docker build -t crawler .
+	@echo -ne "Building crawler container..."
+	@docker build -t crawler . 1>/dev/null
+	@if [ $$? -eq 0 ];then echo "[OK]";else echo "[FAILED]"; fi
 
+setupelk:
+	@echo -ne "Building elk container..."
+	@docker build -t crawler-elk -f Dockerfile.elk . 1>/dev/null
+	@if [ $$? -eq 0 ];then echo "[OK]";else echo "[FAILED]"; fi
+	@echo -ne "Starting elk container..."
+	@docker run -d -p 5601:5601 -p 9200:9200 -p 5044:5044 -p 5000:5000 -p 8080:8080 --name elk crawler-elk 1>/dev/null
+	@echo "[OK]"
+	@#sleep is added to allow time for elk components to bootstrap
+	@#before crawler can send data to it
+	@sleep 2
+	@echo -ne "Starting crawler container..."
+	@docker run \
+		--privileged \
+		--net=host \
+		--pid=host \
+		--name agentless-crawler\
+		-v /cgroup:/cgroup:ro \
+		-v /sys/fs/cgroup:/sys/fs/cgroup:ro \
+		-v /var/run/docker.sock:/var/run/docker.sock \
+		-it -d crawler python2.7 crawler.py\
+		 --url "logstash://127.0.0.1:8080/crawler/data"\
+		 --features os,disk,process\
+		 --frequency 5\
+		 --crawlmode OUTCONTAINER\
+		 1>/dev/null
+	@echo "[OK]"
+	@echo "You can view the crawl data at http://localhost:5601/"
```

Comment:
if you hit http://localhost:5601, doesn't it get redirected to http://localhost:5601/app/kibana ?